libs/auth/src/signed_hash_validation.ts

```diff
@@ -91,6 +91,9 @@ export async function generateSignedHashValidationQrCodeValue(
     : '';
   const date = new Date();
 
+  const machineCert = await fs.readFile(config.machineCertPath);
+  const { machineId } = await parseMachineDetailsFromCert(machineCert);
```

Comment:
I don't think you need the `machineId` in the payload anymore for the signed QR code, it's part of the cert.

Reply:
Oh this is a good catch!

Reply:
Noting that this will require an update to https://github.com/votingworks/livecheck-web, to extract machine ID from the cert

Will ticket that to make sure that we don't forget

Reply:
https://github.com/votingworks/vxsuite/pull/5557/commits/a49f1967f2240c563309d693206b5af6d5fccba7

Follow-up issue: https://github.com/votingworks/vxsuite/issues/5575